docfx.json

```diff
@@ -508,11 +508,9 @@
         "docs/core/deploying/**.md": ".NET Core",
         "docs/core/docker/**.md": ".NET Core",
         "docs/core/migration/**.md": ".NET Core",
-        "docs/core/porting/**.md": ".NET Core",
         "docs/core/testing/**.md": ".NET Core",
         "docs/core/tools/**.md": ".NET Core CLI",
         "docs/core/tutorials/**.md": ".NET Core",
-        "docs/core/versions/**.md": ".NET Core",
```

Comment:
The articles under `versions` folder don't need suffix at all. All titles already have ".NET Core".
This is the list of the titles:

- How the .NET Core Runtime and SDK are versioned
- Remove the .NET Core runtime and SDK
- Select which .NET Core version to use